components/PickerBase/PickerBase.content.comp.cy.js

```diff
@@ -174,4 +174,97 @@ describe('Test the default PickerBase content', () => {
           .should('have.class', 'is-invalid');
       });
   });
+
+  it('All button toggles all options when clicked', () => {
+    const readySpy = cy.spy().as('readySpy');
+    const updateSpy = cy.spy().as('updateSpy');
+
+    cy.mount(PickerBase, {
+      props: {
+        onReady: readySpy,
+        'onUpdate:picked': updateSpy,
+        label: 'Picker',
+        options: ['Option 1', 'Option 2', 'Option 3', 'Option 4'],
+        invalidFeedbackText: 'Invalid feedback text.',
+      },
+    });
+
+    cy.get('@readySpy').should('have.been.calledOnce').then(() => {
+
+      cy.get('[data-cy="picker-options"]').find('input').should('not.be.checked');
+      
+      cy.get('[data-cy="picker-all-button"]').should('contain.text', '✅ All');
+
+      cy.get('[data-cy="picker-all-button"]').click();
+
+      cy.get('[data-cy="picker-options"]').find('input').should('be.checked');
+      
+      cy.get('[data-cy="picker-all-button"]').should('contain.text', '🚫 All');
+
+      cy.get('@updateSpy').should('have.been.calledWith', [
```

Comment:
This assertion is not necessary here as it is tested in the `events` tests.  Similar comment applies to other tests here that check the `picked` event payload.